src/frontend/efiling-frontend/src/components/page/home/Home.js

```diff
@@ -1,9 +1,62 @@
-import React from "react";
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import queryString from "query-string";
+import { useLocation } from "react-router-dom";
+import axios from "axios";
+import Header, { Footer, Loader } from "shared-components";
+import CSOStatus from "../../composite/csostatus/CSOStatus";
+
+import "../page.css";
+
+// make call to submission/{id}/userDetail to get the user details
+export const checkCSOAccountStatus = (
+  submissionId,
+  setCsoAccountExists,
+  setShowLoader
+) => {
+  axios
+    .get(`/submission/${submissionId}/userDetail`)
+    .then(({ data: { csoAccountExists } }) => {
+      if (csoAccountExists) setCsoAccountExists(true);
+      setShowLoader(false);
+    })
+    .catch(() => {
+      throw new Error(
+        "An error occurred with getting user details. Please try again."
```

Comment:
Do we need to handle API error handling here?

Reply:
yup, outside the scope of this pr but eventually we will need some sort of error page where we can take the user to under any backend call error state i believe (or something similar, will have to discuss and figure out flow with julia and team first)